packages/sources/superstate/src/config/index.ts

```diff
@@ -0,0 +1,23 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig(
+  {
+    API_ENDPOINT: {
+      description: 'An API endpoint for Superstate',
+      type: 'string',
+      required: true,
+      default: 'https://api.superstate.co/v1',
+    },
+    LOOKBACK_DAYS: {
+      description: 'The number of days of historical data to retrieve',
+      type: 'number',
+      required: true,
+      default: 10,
+    },
+  },
```

Comment:
Should this be an input param rather than env config? Looks like we want all nodes to be using the same lookback period

Reply:
As we care only about the last value from returned array, the lookback period does not have to be the same for all nodes. In most of the cases `LOOKBACK_DAYS: 2` and  `LOOKBACK_DAYS: 100` will result to the same value. I think it's more important here to set some minimum acceptable options. My question is why we need to make this input param then ? So that we can set it in the RDD config and that way be sure that it is the same for all nodes ?

Reply:
Ah I see - so we just need `LOOKBACK_DAYS` to be greater than the number of days that a report may have not been generated (weekends + holidays)?

Original suggestion for input params was so that the job spec defines the lookback rather than environment config,  but it sounds like that doesn't affect the actual data returned by the EA so it's not an issue - env config sounds fine 👍